packages/Alert/src/alert.scss

```diff
@@ -261,13 +255,14 @@
 
     &__icon {
         font-size: 0;
+        display: none;
 
         + .ids-alert__body {
-            margin-left: var(--ids-alert-flex-gap-component);
+            margin-left: 0;
         }
 
         @media (width >= #{tokens.$breakpoints-sm}) {
```

Comment:
I'm guessing when I updated the way the breakpoints were written, this used to be <= and I wrote it wrong, but the way you did it works too.

Reply:
I can also only flip the > to <. I tend to go mobile first.

Reply:
yes mobile first is the way to go. Some were just done the quick way I think... Not a huge deal in this case.